### PR TITLE
Fix #245: Improved Comments section of Story page

### LINKED ISSUE #245: [Feature] Improve the Comments section in Story page
## Is your feature request related to a problem? Please describe.

Currently the comments section is not well-decorated or organized. 

image

## Describe the solution you'd like

The comments section should be organized according to the new design. 
image

### PR DESCRIPTION
**Pull request by kailash360**
**Issue Number**

fixes #245 

**Describe the changes you've made**

I have restructured the JSX of the `comments.js` and `commentForm.jsx` component to fit according to the new layout. I have also modified the stylesheets to enhance the UI of the same. I have also modified the UI in case no comment is found.

**Describe if there is any unusual behavior (Any Warning) of your code(Write `NA` if there isn't)**

NA

**Checklist**

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] The title of my pull request is a short description of the requested changes.

**Provide a Deployed link of route/page that needs to review**

Preview: Deploy preview link here with the appropriate route

https://user-images.githubusercontent.com/75155230/179371495-875818ba-7a78-47e1-8019-452c77558303.mp4

### CODE REVIEW

**Comment by mharshita** [comment]
Facing this bug for comment replies

https://user-images.githubusercontent.com/60281688/179404626-b0b21b96-e70d-4403-9769-78a3ee5e5838.mov

File: src/assets/scss/components/comments.scss [hunk 1]
```diff
@@ -3,44 +3,86 @@
...
+
 }
 
 .comment {
   display: flex;
   flex-direction: row;
   margin: 8px 0px;
+
+  .avatar {
+    height: fit-content;
```
Comment (mharshita) [inline src/assets/scss/components/comments.scss:20]: This is giving quite a weird look to the profile pictures. I guess setting height as 2.5rem too will fix.
Screenshot 2022-07-17 at 9 15 05 PM

**Comment by mharshita** [comment]
Similarly can you fix the avatar on the heading too ?
Screenshot 2022-07-17 at 9 31 13 PM

File: src/assets/scss/components/comments.scss (hunk 1 above)
Comment (kailash360) [reply src/assets/scss/components/comments.scss]: Done

**Comment by kailash360** [comment]
@mharshita 
I have pushed the changes

The bug is fixed now.
https://user-images.githubusercontent.com/75155230/179423082-9df7ae52-7b9b-4680-a510-80240d5a5d1e.mp4

**Review by mharshita — APPROVED** [review]

**Pull request merged**